openstack/datadog_checks/openstack/openstack.py

```diff
@@ -794,6 +797,8 @@ def check(self, instance):
             # Store the scope on the object so we don't have to keep passing it around
             self._current_scope = instance_scope
 
+            collect_all_projects = instance.get("collect_all_projects", False)
+
```

Comment:
any reason not to default to true?

Reply:
my thinking here was to default to False since the check collects ~ 20 metrics per project, which would increase the number of metrics collected per check run